helpers/util.sh

```diff
@@ -70,6 +70,10 @@ LOCAL_REPO=$ANDROID_ROOT/droid-local-repo/$DEVICE
 mkdir -p $LOCAL_REPO
 PLUS_LOCAL_REPO="--plus-repo $LOCAL_REPO"
 
+if [ -z $HABUILD_TARGET ] ; then
```

Comment:
Should we check for that or ask the to take care of this? The SFOS version isn't directly an indicator of breaking changes, it can but it doesn't have too.
Often breaking changes are in this repo or droid-configs and its dependencies.